app/src/main/java/com/epfl/beatlink/ui/search/SearchBarScreen.kt

```diff
@@ -0,0 +1,128 @@
+package com.epfl.beatlink.ui.search
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.epfl.beatlink.R
+import com.epfl.beatlink.ui.navigation.BottomNavigationMenu
+import com.epfl.beatlink.ui.navigation.LIST_TOP_LEVEL_DESTINATION
+import com.epfl.beatlink.ui.navigation.NavigationActions
+import com.epfl.beatlink.ui.search.components.ShortSearchBarLayout
+import com.epfl.beatlink.ui.search.components.StandardLazyColumn
+import com.epfl.beatlink.ui.theme.PrimaryPurple
+import com.epfl.beatlink.ui.theme.PrimaryRed
+
+@Composable
+fun SearchBarScreen(navigationActions: NavigationActions) {
+  val selectedCategory = remember { mutableStateOf("Songs") }
+  val recentSearches = remember { mutableStateOf(listOf("Song 1")) }
+
+  Scaffold(
+      topBar = { ShortSearchBarLayout(navigationActions) },
+      bottomBar = {
+        BottomNavigationMenu(
+            onTabSelect = { route -> navigationActions.navigateTo(route) },
+            tabList = LIST_TOP_LEVEL_DESTINATION,
+            selectedItem = navigationActions.currentRoute())
+      },
+      modifier = Modifier.testTag("searchBarScreen")) { paddingValues ->
+        Column(
+            modifier =
+                Modifier.testTag("recentSearchesColumn")
+                    .fillMaxSize()
+                    .padding(paddingValues)
+                    .background(color = Color.White)) {
```

Comment:
here you should use the app backgroup (eg. lightThemeBackground) instead of plain white